src/accounts/SavingAccount.java

```diff
@@ -0,0 +1,51 @@
+package accounts;
+
+import java.math.BigDecimal;
+
+public class SavingAccount extends Account {
+    private BigDecimal goalAmount;
+
+    public SavingAccount(String accountType, String accountNum, String owner, BigDecimal amount, BigDecimal goalAmount) {
+        super(accountType, accountNum, owner, amount);
+        this.goalAmount = goalAmount;
+    }
+
+    @Override
+    public String getAccountInfo() { // 계좌 정보 출력
+        return super.getAccountInfo() + "\n" +
+                "Goal Asset : ₩" + goalAmount;
+    }
+
+    @Override
+    public boolean withdraw(BigDecimal withdrawAmount) { // 출금
+        if(isValid(withdrawAmount)) {
```

Comment:
```suggestion
        if(!isValid(withdrawAmount)) {
           System.out.println("WithDraw Failed your Amount : ₩" + amount); // 예산 < 출금 금액
           return false;
        }
```
if문의 조건을 반대로 설정함으로써 반복 중첩문을 줄일 수 있을 것 같아요